FirebasePerformance/Sources/Instrumentation/FPRNetworkTrace.m

```diff
@@ -191,9 +200,9 @@ - (void)start {
     [self checkpointState:FPRNetworkTraceCheckpointStateInitiated];
 
     FPRSessionManager *sessionManager = [FPRSessionManager sharedInstance];
-    [self updateTraceWithCurrentSession];
+    [self updateTraceWithCurrentSession:[sessionManager sessionDetails]];
```

Comment:
Why is it safe to access `[sessionManager sessionDetails]` here? Should we copy it before passing to the method? 

The same questions to similar changes in other traces.

Reply:
Good point. Makes sense. Updated it.